src/reducers/flavor.js

```diff
@@ -40,7 +67,12 @@ export const reducer = (state = initialState, action = {}) => {
         stash: action.stash,
         loaded: true
       };
-    case types.REQUEST_STASH_FAILURE:
+    case types.UPDATE_STASH_SUCCESS:
+      return {
+        ...state,
+        loaded: false
+      };
+    case types.REQUEST_FAILURE:
```

Comment:
I think this is fine, but can you think of any instance where we'd need to know _which_ request failed?

Reply:
This was more of a trial, but there is/was only one `error` property. Maybe I'm not seeing the bigger picture, but there was nothing to distinguish which request failed previously.